sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -834,21 +819,9 @@ class SessionCatalog(
     }
   }
 
-  private def getTempViewPlan(plan: LogicalPlan): LogicalPlan = {
-    plan match {
-      case TemporaryViewRelation(tableMeta, None) =>
-        fromCatalogTable(tableMeta, isTempView = true)
-      case TemporaryViewRelation(tableMeta, Some(plan)) =>
-        View(desc = tableMeta, isTempView = true, child = plan)
-      case other => other
-    }
-  }
-
-  def getTempViewSchema(plan: LogicalPlan): StructType = {
```

Comment:
This is only called from `SparkGetColumnsOperation.scala` and now that a raw temp plan always has `CatalogTable`, the caller can just directly get the schema from the raw plan, and I don't see a reason to expose this as a `def`. Check `SparkGetColumnsOperation.scala` changes below.